website/layouts/index.html

```diff
@@ -52,7 +52,33 @@ <h4 class="font-weight-bold">Downsampling & Compaction</h4>
         </div>
     </div>
 </div>
-<div class="container-fluid">
+<div class="container py-5">
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Created by</h3></div>
+    </div>
+    <div class="row">
+        <div class="col-4 col-xl-2 offset-4 offset-xl-5 py-3">
+            <div class="img-sponsor">
+                <a href="https://improbable.io" rel="nofollow" target="_blank"><img src="{{ "/logos/improbable.png" | relURL }}" class="img-fluid" alt="Improbable"></a>
+            </div>
+        </div>
+    </div>
+    {{ if $.Site.Data.sponsors.sponsors }}
+    <div class="row">
+        <div class="col-12"><h3 class="text-center">Used by</h3></div>
+    </div>
+    <div class="row">
+        {{ range $sponsor := $.Site.Data.sponsors.sponsors }}
+        <div class="col-4 col-xl-2 py-3">
+            <div class="img-sponsor">
+                <a href="{{ $sponsor.url }}" rel="nofollow" target="_blank"><img src="{{ (printf "/logos/%s" $sponsor.logo) | relURL }}" alt="{{ $sponsor.name }}"></a>
```

Comment:
what will happen if the logo is not present? Maybe have a default Thanos image to put there?

Reply:
Added a check for the logo property !